### PR TITLE
Fix local run without INTEGRATION_KEY #8

### PR DESCRIPTION
**Pull request by maoo**
Fixes issue #7 

The implementation is not very clean, but it does the trick; feel free to point out better implementations

### CODE REVIEW

**Comment by ColinEberhardt** [comment]
Thanks @maoo - I think a simpler way to fix the issue is to move the initialisation code into the body of the function. This only gets executed if `INTEGRATION_ENABLED` is true.

I've not tried it, but something like this should work: 

```
const fs = require('fs');
const jwt = require('jsonwebtoken');
const requestp = require('./requestAsPromise');

const integrationId = process.env.INTEGRATION_ID;

module.exports = (installationId) => {
  const cert = fs.readFileSync(process.env.INTEGRATION_KEY);
  const token = jwt.sign({ iss: integrationId },
    cert, {
      algorithm: 'RS256',
      expiresIn: '10m'
    });

  return requestp({
    url: `https://api.github.com/installations/${installationId}/access_tokens`,
    json: true,
    headers: {
      'Authorization': 'Bearer ' + token,
      'User-Agent': 'github-cla-bot',
      'Accept': 'application/vnd.github.machine-man-preview+json'
    },
    method: 'POST'
  })
  .then(({token}) => token);
};
```

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,7 @@
 language: node_js
 node_js:
 - '6'
-before_install:
-- openssl aes-256-cbc -K $encrypted_369f74a373d5_key -iv $encrypted_369f74a373d5_iv
-  -in secrets.tar.enc -out secrets.tar -d
-- tar xvf secrets.tar
+before_install: "if [[ \"$TRAVIS_PULL_REQUEST\" = \"false\" ]]; then openssl aes-256-cbc -K $encrypted_369f74a373d5_key -iv $encrypted_369f74a373d5_iv -in secrets.tar.enc -out secrets.tar -d ; tar xvf secrets.tar ; fi"
```
Comment (ColinEberhardt) [inline .travis.yml]: Ah - this looks useful. Could you raise a separate PR with that fix in it? Thanks!

Comment (maoo) [reply .travis.yml]: reverted change, I'll send a new PR as soon as the rest is merged

**Comment by ColinEberhardt** [comment]
@maoo - are you planning to tidy up this PR for merge? it's a useful change :-)

**Comment by maoo** [comment]
Yup, here it is, closing this one - https://github.com/ColinEberhardt/cla-bot/pull/12

File: .travis.yml (hunk 1 above)
Comment (maoo) [reply .travis.yml]: Here it is! https://github.com/ColinEberhardt/cla-bot/pull/13

**Pull request closed without merging**